src/controllers/albums.js

```diff
@@ -0,0 +1,110 @@
+const db = require('../../src/db/index.js');
+
+const createAlbum = async (req, res) => {
+  const { name, date } = req.body;
+  const { id } = req.params;
+
+  try {
+    const {
+      rows: [album],
+    } = await db.query(
+      'INSERT INTO Albums (name, date, artistid) VALUES ($1, $2, $3) RETURNING *',
+      [name, date, id]
+    );
+    res.status(201).json(album);
+  } catch (err) {
+    res.status(500).json(err.message);
+  }
+};
+
+const readAlbum = async (req, res) => {
+  try {
+    const { rows } = await db.query('SELECT * FROM Albums');
+    res.status(200).json(rows);
+  } catch (err) {
+    res.status(500).json(err.message);
+  }
+};
+
+const getAlbumById = async (req, res) => {
+  try {
+    const { id } = req.params;
+    const {
+      rows: [album],
+    } = await db.query('SELECT * FROM Albums WHERE id = $1', [id]);
+
+    if (!album) {
+      res.status(404).json({ message: `Albums ${id} does not exist` });
+    }
+    res.status(200).json(album);
+  } catch (err) {
+    res.status(500).json(err.message);
+  }
+};
+
+const updateAlbum = async (req, res) => {
+  const { id } = req.params;
+  const { name, date, artistid } = req.body;
+
+    let query;
+    let params;
+
+  if (name && date && artistid) {
```

Comment:
The 2 `let` statements here dont need any further indentation:
```
  const { name, date, artistid } = req.body;

  let query;
  let params;

  if (name && date && artistid) {
```

You could also group these together on one line: 
```
  let query, params;
```